client/js/widgets/button.js

```diff
@@ -141,7 +141,11 @@ class Button extends Widget {
         const count = a.count || 999999;
 
         if(isValidID(a.from) && isValidID(a.to)) {
-          if(a.face === null && typeof a.from == 'string' && typeof a.to == 'string' && !widgets.get(a.to).children().length && widgets.get(a.from).children().length <= count) {
+          const item = widgets.get(a.from)
+          const dest = widgets.get(a.to)
+          if (item && item.p('type') == null && dest.p('type') == 'holder') {
+            item.moveToHolder(dest);
```

Comment:
You only do the `moveToHolder`; not the `flip`, not the `bringToFront` or the `parent`. That's probably what LawDawg noticed.

Maybe find a better way to not duplicate the code? Easiest would probably to move the function inside the `forEach` to the top of the `MOVE(XY)` blocks and call that with a single widget?

Reply:
fixed